pylab/website/templates/website/project_details.html

```diff
@@ -14,9 +14,20 @@
   {% endif %}
 </div>
 
-<h1>{{ project.title }}</h1>
+<h1>{{ project.title }}
+  <small><span class="label label-default">{{ project.get_status_display }}</span></small>
+</h1>
 
-{{ project.description|markdown }}
+<p><strong>Project description </strong>{{ project.description|markdown }}</p>
```

Comment:
Strings should be marked as translatable in templates. You can do it by using ```trans``` templatetag. At the beginning of the template import it ```{% load trans blocktrans from i18n %}``` and mark string as translatable: ```{% trans "Project description" %}```.